client/src/pages/HomePage/HomePage.js

```diff
@@ -1,10 +1,64 @@
-import React from 'react';
+import React, { useContext, useState, useEffect } from 'react';
+import axios from 'axios';
+import { StateContext } from '../../App';
+import { Box, Typography } from '@mui/material';
+import AnimalTypeBox from './components/AnimalTypeBox';
+import AddAnimal from './components/AddAnimal';
+import {
+  homePageContainer,
+  animalTypeContainer,
+  titleText,
+} from '../../styles/home/animalTypeBoxStyles';
 
 const HomePage = () => {
+  const { state } = useContext(StateContext);
+  const [animalTypes, setAnimalTypes] = useState([]);
+
+  const axiosConfig = {
+    headers: {
+      Authorization: `Bearer ${state.token}`,
+    },
+  };
+
+  const fetchData = async () => {
+    try {
+      const response = await axios.get(
+        'http://localhost:5000/api/v1/animals',
+        axiosConfig
+      );
+      const data = response.data.animalList;
```

Comment:
should be 
`const data = response.data`
`const animalList = data.animalList`

Reply:
i know for now this looks trivial but imagine if daghan ug stuff ang data, how will you name na them?
this way sad you seperate the actual `data` of the api response

Reply:
resoved